### DISCUSSION TITLE
Delete "Browsers" and add new ones

### DISCUSSION THREAD

**Discussion by ElSui** [will-stone/browserosaurus#497] [Ideas]
I'm sorry if this is already a request. I tried looking around but haven't found it. 

I would love to be able to delete elements from the list of browsers that are proposed by Browserosaurus. I have some elements on the list that are NEVER a relevant use case for me. 

Also, it would be awesome to be able to add "browsers" to that list. Like Notion for example. Notion is a tool that I use all day every day, and opening links directly into it would be amazing. 

Cheers, and thank you for the incredible work! 🙏

**Comment by will-stone** [comment]
Hi @ElSui, firstly thanks for using Browserosaurus. I'm glad you find it useful. I'm not a fan of hiding browsers as I simply don't think it's necessary with the current UI; your unused browsers can be moved to end of the list, and the picker window kept small enough to hide them. As for supporting new browsers, please see [this document](https://github.com/will-stone/browserosaurus/blob/main/docs/supporting-a-browser-or-app.md).

**Reply by ElSui** [reply]
Hi @will-stone, thank you for your reply. I will be taking a look at that document and add the browsers myself when I get some time. I'll be adding Notion and [Around](https://www.around.co).

Regarding the ability to delete browsers, I get your point and as you take the time to build the tool I have to respect it. Nevertheless, I think we were able to do some type of deeper customization in previous versions when the picker window was displaying browsers in a horizontal line.

I can work around the inability to delete browsers, don't worry, it's just my product geek maniac eye that is very picky. 😅

Cheers and thank you!

**Comment by richardbrockie** [comment]
I for one would also like a way to prune the browser list. For example, Zoom is being offered by Browserosaurus on my systems and I would like to remove it.

**Reply by andoro** [reply]
The same opinion from me. For example why I'd like to open anything in Spotify, Slack, Figma?

**Comment by mknotzer** [comment]
checkboxes would be great

<img width="843" alt="image" src="https://github.com/will-stone/browserosaurus/assets/13821046/588d8197-b488-4c8a-870f-c744979e6045">

The goal is to get rid of the scrollbar

<img width="242" alt="image" src="https://github.com/will-stone/browserosaurus/assets/13821046/e5a46396-bc43-4eeb-b964-0341c78ba398">
